Domains/2-DisciplinePhysical/Civil/SpatialComposition/CivilSpatial/CivilSpatial.ecschema.xml

```diff
@@ -0,0 +1,135 @@
+        <Source multiplicity="(0..*)" roleLabel="is of" polymorphic="true">
+            <Class class="ParkingArea" />
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is type of" polymorphic="true">
+            <Class class="ParkingAreaType"/>
+        </Target>
+    </ECRelationshipClass>
+    <ECEntityClass typeName="ParkingIsland" modifier="None" displayLabel="Parking Island" description="A cvsp:Space modeling a subsidiary area raised or marked on a cvsp:ParkingArea, shaped and placed so as to direct traffic movement and/or provide refuge for pedestrians.">
+        <BaseClass>Space</BaseClass>
+    </ECEntityClass>
+    <ECEntityClass typeName="ParkingIslandType" displayLabel="Parking Island Type" description="Defines a shared set of properties whose values vary per-type of cvsp:ParkingIsland rather than per-instance.">
+        <BaseClass>spcomp:SpaceType</BaseClass>
+    </ECEntityClass>
+    <ECRelationshipClass typeName="IslandAreaIsOfType" strength="referencing" modifier="None" description="A type-instance relation; one that indicates that the specific cvsp:ParkingIsland is an instance of the defined cvsp:ParkingIslandType.">
+        <BaseClass>spcomp:SpaceIsOfType</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="is of" polymorphic="true">
+            <Class class="ParkingIsland" />
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is type of" polymorphic="true">
+            <Class class="ParkingIslandType"/>
+        </Target>
+    </ECRelationshipClass>
+    <ECEntityClass typeName="ParkingRow" modifier="None" displayLabel="Parking-Row" description="A cvsp:Space modeling a location that is designated for parking of one or more vehicles.">
+        <BaseClass>Space</BaseClass>
+        <ECProperty propertyName="ParkingSpaceCount" typeName="int" displayLabel="Parking-Space Count" description="Number of parking-spaces in this Parking-Row"/>
+    </ECEntityClass>
+    <ECEntityClass typeName="ParkingRowType" displayLabel="Parking-Row Type" description="Defines a shared set of properties whose values vary per-type of cvsp:ParkingRow rather than per-instance.">
+        <BaseClass>spcomp:SpaceType</BaseClass>
+    </ECEntityClass>
+    <ECRelationshipClass typeName="ParkingRowIsOfType" strength="referencing" modifier="None" description="A type-instance relation; one that indicates that the specific cvsp:ParkingRow is an instance of the defined cvsp:ParkingRowType.">
+        <BaseClass>spcomp:SpaceIsOfType</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="is of" polymorphic="true">
+            <Class class="ParkingRow" />
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is type of" polymorphic="true">
+            <Class class="ParkingRowType"/>
+        </Target>
+    </ECRelationshipClass>
+    <ECEntityClass typeName="Sidewalk" displayLabel="Sidewalk" description="A cvsp:Space modeling a footpath for pedestrians.">
```

Comment:
Some class definitions do not specify `modifier` attribute (like Sidewalk) while most other do. Perhaps we should specify it for every class for consistency?